### PR TITLE
Folders Added: Assignment 4 & 5

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

File: akash1/JavaScript/ASSIn4/Ass4.1.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
+            box-shadow: 0 10px 12px 0 rgba(213, 215, 216, 0.973), 0 6px 20px 0 rgba(206, 14, 14, 0.918);
+             top: 149px;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.1.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.1.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+             position: relative;
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.1.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.1.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.1.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.2.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.2.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.3.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.3.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.4.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.4.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn4/Ass4.5.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn4/Ass4.5.html:19]: wrong Indentation.

File: akash1/JavaScript/ASSIn5/Ass5.1.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn5/Ass5.1.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn5/Ass5.2.html [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn5/Ass5.2.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn5/Ass5.3.html [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn5/Ass5.3.html]: wrong Indentation.

File: akash1/JavaScript/ASSIn5/Ass5.4.html [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+            integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l"
+            crossorigin="anonymous"
+        />
+        <style>
+            body{
+                background-image: linear-gradient(black,skyblue, black);
+                background-repeat: no-repeat;
+                height: 100vh;
+            }
+        .container{
```
Comment (Pranjali1999) [inline akash1/JavaScript/ASSIn5/Ass5.4.html]: wrong Indentation.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

**Review by vinod2406 — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**